intro-HPC/antwerpen/macros.tex

```diff
@@ -3,14 +3,14 @@
 \macro{\university}{University of Antwerp}
 \macro{\association}{Antwerp University Association (AUHA)}
 % HPC definitions
-\macro{\hpc}{UA-HPC}
+\macro{\hpc}{UAntwerpen-HPC}
 \macro{\hpcname}{hopper}
 \macro{\loginnode}{login.hpc.uantwerpen.be}
 \macro{\loginhost}{ln02.hopper.uantwerpen.vsc}
 %TODO: RSA key fingerprint here
-\macro{\loginfingerprint}{b7:66:42:23:5c:d9:43:e8:b8:48:6f:2c:70:de:02:eb}
-\macro{\operatingsystem}{Scientific Linux release 6.4 (Carbon)}
-\macro{\operatingsystemSL}{Scientific Linux}
+\macro{\loginfingerprint}{20:96:ca:e7:f0:62:87:34:ca:9c:71:84:b7:d3:46:f8}
+\macro{\operatingsystem}{Scientifc Linux release 6.6 (Carbon)}
```

Comment:
You’re right. The version number had also changed, which drew my attention rather than the typo in Scientific. I’ve pushed a correction to the repository.

Regards,

Kurt

Van: Jens Timmerman <notifications@github.com<mailto:notifications@github.com>>
Beantwoorden - Aan: hpcugent/vsc_user_docs <reply@reply.github.com<mailto:reply@reply.github.com>>
Datum: donderdag 5 februari 2015 11:52
Aan: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com<mailto:vsc_user_docs@noreply.github.com>>
CC: Kurt Lust <Kurt.Lust@uantwerpen.be<mailto:Kurt.Lust@uantwerpen.be>>
Onderwerp: Re: [vsc_user_docs] Change of names for the University of Antwerp. (#94)


In intro-HPC/antwerpen/macros.tex<https://github.com/hpcugent/vsc_user_docs/pull/94#discussion_r24155818>:

>  \macro{\hpcname}{hopper}
>  \macro{\loginnode}{login.hpc.uantwerpen.be}
>  \macro{\loginhost}{ln02.hopper.uantwerpen.vsc}
>  %TODO: RSA key fingerprint here
> -\macro{\loginfingerprint}{b7:66:42:23:5c:d9:43:e8:b8:48:6f:2c:70:de:02:eb}
> -\macro{\operatingsystem}{Scientific Linux release 6.4 (Carbon)}
> -\macro{\operatingsystemSL}{Scientific Linux}
> +\macro{\loginfingerprint}{20:96:ca:e7:f0:62:87:34:ca:9c:71:84:b7:d3:46:f8}
> +\macro{\operatingsystem}{Scientifc Linux release 6.6 (Carbon)}


is it really not Scientific ?

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/94/files#r24155818>.